src/components/ui/Chat/ChannelList/ChannelItem.tsx

```diff
@@ -22,25 +22,80 @@ export interface ChannelItemProps
   isSelected?: boolean;
   /** Callback function when clicked */
   onClick: () => void;
-  /** Signifies of there are unread message in the channel. */
+  /** Signifies if there are unread message in the channel. */
   unread?: boolean;
   /** Content of the badge signaling unread messages, such as a badge displaying the number. */
   unreadBadgeLabel?: string;
+  /** Indicate if we want to display the detailed channel view with last channel message or timestamp */
+  displayDetailedChannelView?: boolean;
+  /* Content of the last channel message to be displayed */
+  lastChannelMessage?: string;
+  /* Timestamp of the last channel message to be displayed */
+  lastChannelMessageTimestamp?: string;
```

Comment:
I just noticed that the `timestamp` prop in the component library has a string type. Usually, the timestamp should have either `Date` or `number`, and the SDK should decide which format to display.